crates/bevy_reflect/src/impls/std.rs

```diff
@@ -662,6 +656,231 @@ crate::func::macros::impl_function_traits!(::bevy_utils::hashbrown::HashMap<K, V
     >
 );
 
+macro_rules! impl_reflect_for_hashset {
+    ($ty:path) => {
+        impl<V, S> Set for $ty
+        where
+            V: FromReflect + TypePath + GetTypeRegistration + Eq + Hash,
+            S: TypePath + BuildHasher + Send + Sync,
+        {
+            fn get(&self, value: &dyn Reflect) -> Option<&dyn Reflect> {
+                value
+                    .downcast_ref::<V>()
+                    .and_then(|value| Self::get(self, value))
+                    .map(|value| value as &dyn Reflect)
+            }
+
+            fn get_mut(&mut self, value: &dyn Reflect) -> Option<&mut dyn Reflect> {
+                value
+                    .downcast_ref::<V>()
+                    .and_then(|value| Self::get_mut(self, value))
```

Comment:
Are we sure this is not just recursing into this exact method? AFAIK types like `HashSet` don't have a `get_mut` method.

Reply:
Yeah makes total sense. I even noted this down in the PR description at some point 🤦🏼 I just removed the method again